fitbenchmarking/utils/create_dirs.py

```diff
@@ -47,26 +47,19 @@ def group_results(results_dir, group_name):
     return group_results_dir
 
 
-def restables_dir(results_dir, group_name):
+def restables_dir(results_dir):
     """
     Creates the results directory where the tables are located.
     e.g. fitbenchmarking/results/neutron/
 
-    @param results_dir :: directory that holds all the results
-    @param group_name :: string containing the name of the problem group
+    @param results_dir :: directory that holds all the results for the problem
```

Comment:
Shall we update the doc strings for this fill at the same time?

Reply:
I'll do this in a different issue as I've already started there